lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -41,10 +41,11 @@
             EmailFieldView: FieldViews.TextFieldView.extend({
                 fieldTemplate: field_text_account_template,
                 successMessage: function () {
-                    return this.indicators.success + StringUtils.interpolate(
+                    return HtmlUtils.joinHtml(
+                        this.indicators.success, StringUtils.interpolate(
```

Comment:
I assume `indicators.success` contains an HTML tag.  If it isn't too large of a project, I'd prefer `success` to be named `successHtml`. 
Also, could you put `StringUtils.interpolate()` param on the next line, indented with the first parameter?